### PR TITLE
Chore: use eslintrc overrides

### PR DESCRIPTION
**Pull request by Aladdin-ADD**
**What is the purpose of this pull request? (put an "X" next to item)**

[ ] Documentation update
[ ] Bug fix ([template](https://raw.githubusercontent.com/eslint/eslint/master/templates/bug-report.md))
[ ] New rule ([template](https://raw.githubusercontent.com/eslint/eslint/master/templates/rule-proposal.md))
[ ] Changes an existing rule ([template](https://raw.githubusercontent.com/eslint/eslint/master/templates/rule-change-proposal.md))
[ ] Add autofixing to a rule
[ ] Add a CLI option
[ ] Add something to the core
[x] Other, please explain:

**What changes did you make? (Give an overview)**
similar to https://github.com/eslint/eslint/pull/10566

**Is there anything you'd like reviewers to focus on?**
no

### CODE REVIEW

File: packages/eslint-config-eslint/default.yml [hunk 1]
```diff
@@ -1,10 +1,6 @@
 extends:
     - "eslint:recommended"
     - "plugin:node/recommended"
-
-plugins:
-    - "node"
```
Comment (platinumazure) [inline packages/eslint-config-eslint/default.yml]: Why are you removing the Node plugin here? It seems we should specify it if we're going to extend from `"plugin:node/recommended"`.

Comment (Aladdin-ADD) [reply packages/eslint-config-eslint/default.yml]: it has been added in `node/recommended`:
https://github.com/mysticatea/eslint-plugin-node/blob/master/lib/configs/recommended.json#L13

Comment (platinumazure) [reply packages/eslint-config-eslint/default.yml]: Oh, weird. I thought the plugins entry was necessary in order to reference the plugin config. If it's not required and everything works fine, we could remove it here.

Comment (Aladdin-ADD) [reply packages/eslint-config-eslint/default.yml]: I created another PR(#10685 ), since it seems not related to this PR.

**Pull request merged**